examples/vite-demo-vanilla-bundle/public/i18n/fr.json

```diff
@@ -100,5 +100,7 @@
   "TITLE": "Titre",
   "TITLE.NAME": "Nom du Titre",
   "TRUE": "Vrai",
-  "X_DAY_PLURAL": "{{x}} journée{{plural}}"
+  "X_DAY_PLURAL": "{{x}} journée{{plural}}",
+  "RBE_BTN_UPDATE": "Mettre à jour la ligne actuelle",
+  "RBE_BTN_CANCEL": "Annuler la ligne actuelle"
```

Comment:
this is just a French thing but I think it would be better with `ligne courante` because `ligne actuelle` actually means `actual line` which is not exactly the best to use. I know it's Google Translate result that you used but I think it's wrong hehe. BTW, there's 2 of them to change 


![image](https://github.com/ghiscoding/slickgrid-universal/assets/643976/bd9315f4-53d0-4f89-b728-1024c491719d)